Pod/Classes/SEGAmplitudeIntegration.m

```diff
@@ -95,9 +95,17 @@ - (void)realTrack:(NSString *)event properties:(NSDictionary *)properties integr
 {
     NSDictionary *options = integrations[@"Amplitude"];
     NSDictionary *groups = [options isKindOfClass:[NSDictionary class]] ? options[@"groups"] : nil;
-    if (groups && [groups isKindOfClass:[NSDictionary class]]) {
+    bool outOfSession = options[@"outOfSession"];
```

Comment:
See the check for groups above and how it is retrieved. We need the same here.

Reply:
Doesn't `[options isKindOfClass:[NSDictionary class]] ? options[@"outOfSession"] : false;` always return `true` if there are integration settings passed in though?